plugins/GSdx/GSRendererHW.cpp

```diff
@@ -186,6 +186,40 @@ void GSRendererHW::SetGameCRC(uint32 crc, int options)
 	GSRenderer::SetGameCRC(crc, options);
 
 	m_hacks.SetGameCRC(m_game);
+
+	// Code for Automatic Mipmapping. Relies on game CRCs.
+	if (theApp.GetConfigI("mipmap_hw") == -1) {
```

Comment:
Enum over here too, Either cast ``theApp.GetConfigI("mipmap_hw")`` to ``HWMipmap`` and do the check with ``HWMipmapLevel::Automatic``.

```
static_cast<HWMipmapLevel>(theApp.GetConfigI("mipmap_hw")) == HWMipmapLevel::Automatic
```

**(or)**

Do it the opposite way by casting the enum to integer. Your wish but the idea is to avoid magic numbers and use defined constants.